DependencyInjection/Compiler/ExceptionWrapperHandlerPass.php

```diff
@@ -27,7 +28,7 @@ public function process(ContainerBuilder $container)
             return;
         }
 
-        if (interface_exists('JMS\Serializer\Handler\SubscribingHandlerInterface')) {
+        if (interface_exists(SubscribingHandlerInterface::class)) {
```

Comment:
The issue with introducing changes like this right now might be that merges between branches might become trickier so we should be careful with doing them.